### ISSUE TITLE
[FormsySelect] 'required' is not being enforced

### ISSUE THREAD

**Issue by mbrookes** [mbrookes/formsy-material-ui#5]
It does correctly affect canSubmit state, but not the field itself.

**Comment by quangdng** [comment]
Is there any progress on this? This bug is pretty annoying.

**Comment by mbrookes** [comment]
@quangdng - can you remind me how this manifests? It's a long time since I've looked at it - I should have been more verbose in the issue description!

**Comment by simpixelated** [comment]
@mbrookes see #28:

>Apparently, setting the required attribute on a FormsyText reports the form state as invalid (the submit button can be disabled), but no message appears below the field? Is it the expected behavior? Shouldn't we do something with the showRequired function?

After some testing with formsy-material-ui@0.4.3, formsy-react@0.18.0, and material-ui@0.15.0, I see that `FormsyText` doesn't display `validationError` unless there is a value in the field, which means `required` and even `validations="isExisty"` don't display an error on the field.

example:
```js
<FormsyText
  name="first"
  floatingLabelText="First name"
  fullWidth={true}
  validations="isExisty"
  validationError="First name is required"
  required
/>
```

The `validationError` never displays, even after invalid submit.

**Comment by simpixelated** [comment]
I actually think this is an issue in formsy-react. See https://github.com/christianalfoni/formsy-react/issues/241

Basically `getErrorMessage` returns null for `required`. If it returned the `validationError`, this library would work fine. Looks like there's some workarounds on that link, so will continue to dig into it.

**Comment by rojobuffalo** [comment]
Closed by #105
